scripts/mgear/maya/shifter/__init__.py

```diff
@@ -341,53 +317,51 @@ def finalize(self):
                 pm.displayInfo("Importing Skin")
                 skin.importSkin(self.options["skin"])
 
-            except:
-                pm.displayWarning("Skin doesn't exist or is not correct. "+self.options["skin"]+" Skipped!")
-
+            except RuntimeError:
+                pm.displayWarning(
+                    "Skin doesn't exist or is not correct. " +
+                    self.options["skin"] + " Skipped!")
 
     def addCtl(self, parent, name, m, color, icon, **kwargs):
-        """
-        Create the control and apply the shape, if this is alrealdy stored
+        """Create the control and apply the shape, if this is alrealdy stored
```

Comment:
First line of a docstring should be a title, or summary. It shouldn't continue on the next line. In Sphinx, this will be used as a header or summary for the subsequent lines, which will become the body.

- [Examples](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)

Reply:
Thanks for the notes.   I will continue tomorrow. :)